massa-storage/src/lib.rs

```diff
@@ -473,6 +505,63 @@ impl Storage {
         }
         Storage::internal_claim_refs(&ids, &mut owners, &mut self.local_used_endorsements);
     }
+
+    /// Store denunciation
+    pub fn store_denunciation(&mut self, denunciation: Denunciation) {
```

Comment:
Is it necessary to have denunciations in the storage ? I think they should just be added to the block header, they are not meant to propagate individually

Reply:
Remove use of Storage in pool